adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -246,14 +246,22 @@ def __init__(
 
     def get_monotonic_time(self) -> float:
         """
-        Provide monotonic time in seconds. Based on underlying implementation
+        Provide monotonic time in nanoseconds. Based on underlying implementation
```

Comment:
This changes the signature of this routine incompatibly. I suggest instead adding a `get_monotonic_time_ns()` routine (upward compatible), or renaming this routine (incompatible, will require a major version bump).

Reply:
The old, imprecise routine will no longer be used inside this module, but we can maintain it if it is called from external modules.

Reply:
If not called anymore, it should be removed. This was never intended to be part of the public API even though it does not start with underscore.